react-page/src/oas_spec/sg/paths/v4.json

```diff
@@ -82,6 +82,103 @@
       }
     }
   },
+  "/user/direct_bank_transfer_payments": {
```

Comment:
`api/v3/user/direct_bank_transfer_payments`

Actually this should be inside `v3.json`